src/test/testing-library/AuthFormFunctions.test.tsx

```
@@ -0,0 +1,371 @@
+          setErrorDetails(error.message);
+        }
+      } else if (error instanceof Error) {
+        setErrorDetails(error.message);
+      } else {
+        setErrorDetails('An unexpected error occurred');
+      }
+    }
+  };
+
+  return { setNotificationType, setErrorDetails, setIsNotificationOpen, mockOnSubmit };
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)